applications/dashboard/models/class.permissionmodel.php

```diff
@@ -70,6 +70,32 @@ public function addDefault($Type, $Permissions, $Junction = null, $JunctionId =
     }
 
     /**
+     * Add the permissions from one permission array to another.
+     *
+     * @param array $perms1 The permissions to be added to.
+     * @param array $perms2 The permissions to add.
+     * @return array Returns an array with all of the permissions in both permissions arrays.
+     */
+    public static function addPermissions($perms1, $perms2) {
```

Comment:
Maybe `($currentPermissions, $newPermissions)` for the method signature? Numbered parameter names seems like bad precedent, no?